js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/kit/TestKit.scala

```diff
@@ -56,15 +56,15 @@ final class TestKit(jsEnv: JSEnv, timeout: FiniteDuration) {
     start(codeToInput(code))
 
   /** Starts a [[Run]] for testing. */
-  def start(input: Input): Run =
+  def start(input: List[Input]): Run =
```

Comment:
It's a bit awkward that we had to change every `Input` into a `List[Input]`. Could we instead keep `Input` as the one and only top type for the the whole input of a JSEnv, and have it contain a `List[InputItem]` or something like that?

Reply:
Yes, the change is a bit awkward. But isn't this the better end state?

IMHO we shouldn't decide on an end state based on how nice the *change* will look like. If you'd like the changes to look nicer, I can offer to add an intermediary commit with the `InputItem` type and then replace it in the next commit. This would separate the functionality from the structural change.

I'm reluctant to introduce a mere wrapper type to a List / Seq since it seems unnecessarily complicated and not what we do in `Linker`. (speaking of which: should we use a `Seq` here?).

Reply:
> But isn't this the better end state?

I'm not completely convinced, but you seem more sure than me about it, so let's take your design.

If we're going to use `Seq`, let's use `scala.collection.immutable.Seq` explicitly.